apps/api/src/persistence/migrations/migrator.ts

```diff
@@ -45,7 +47,9 @@ export class Migrator {
             .join(`\n`);
     }
 
-    async runAllAvailableMigrations(queryRunner: ICoscradQueryRunner): Promise<void> {
+    async runAllAvailableMigrations(queryRunner: ArangoQueryRunner): Promise<void> {
```

Comment:
This is unfortunate. Decide whether we can keep the interface here. We probably won't run migrations on a second db anytime soon, but it's hard to reason about if we muddy the waters.